Makefile

```diff
@@ -10,7 +10,9 @@ CONDA := source $$(conda info --base)/etc/profile.d/conda.sh ; conda activate ;
 	$(CONDA) .conda/docs; \
 	pip install -r trulens_eval/trulens_eval/requirements.txt; \
 	pip install -r trulens_eval/trulens_eval/requirements.optional.txt; \
-	pip install -r docs/docs_requirements.txt
+	pip install -r requirements.docs.txt; \
```

Comment:
what's with all this `; \` stuff?

Reply:
Its for writing one long command composed of multiple commands (separated by ";") over multiple lines (by escaping newline with "\").

Reply:
What difference is it though if you didn't have it? Like either way it'll run each of the commands one line at a time right?

Reply:
I believe by default it does not preserve the environment between each command it launches so each line would have to include the activate conda line.